ddtrace/sampler.py

```diff
@@ -44,8 +45,8 @@
 
 log = get_logger(__name__)
 
-MAX_TRACE_ID = 2 ** 64
-
+# MAX_TRACE_ID is no longer used. It should deprecated and removed.
```

Comment:
should we deprecate now, or leave for later?

Reply:
I rather do it in a follow up. This PR is already getting a bit long. I'll add a `TODO` to the comment.